doc/getting_started.rst

```diff
@@ -17,29 +17,162 @@
 
+.. Edited September 25, 2020 - [USA]TechDude
+ 
 Getting Started
 ===============
 
-OpenShot Video Editor is available on most operating systems (including Linux, Mac, and Windows).
-Visit the official download page and grab the latest and greatest version:
-https://www.openshot.org/download/.
-
-Linux
------
-Depending on which Linux distribution you use, there are likely many ways to install OpenShot. The
-easiest way to ensure you have the very latest version, is to download an AppImage from the
-project website (listed above). Once downloaded, right click on the AppImage file, choose
-Properties, and mark the file as **Executable**. Now, go ahead and double click the
-AppImage to launch it.
-
-Mac
----
-Download the DMG file from the project website (listed above), double click it, and then drag
-the OpenShot application icon into your **Applications** shortcut. This is very similar to how most
-other Mac applications are installed. Now launch OpenShot from Launchpad.
-
-Windows
--------
-Download the Windows installer executable from the project website (listed above), double click it,
-and follow the directions on screen. Once completed, OpenShot will be installed and available
-in your Start menu.
+.. image:: images/getting-started.png
+
+You do not need to be a trained videographer to understand how to create videos 
+well.  Simple editing can keep your viewers engaged longer, and add a 
+professional feel, even if you are not a professional video editor.
+
+Basically, video editing is taking footage, cutting it up, removing the 
+pieces you do not want, and keeping the bits you do.  Back in the old days, 
+editing was slicing reels of film and piecing it together.  Thankfully software 
+makes the whole process much more manageable.
+
+There are three main jobs of video editing:
+
+1.  Remove mistakes or unwanted sections
+2.  Keep the video moving at an engaging pace
+3.  Insert supporting footage, audio, or titles
+
+Use these three points as a checklist as you edit.
+
+Computer
+--------
+Video editing does not require an expensive machine, especially if you are a 
+beginner.  It would be best if you had a decent monitor and graphics card.  If 
+you have an older computer, check your system specifications against OpenShot's 
+`minimum system requirements <introduction.html#system-requirements>`_ 
+to make sure it works for video editing.  Unfortunately, many older computers 
+are not fast enough for video editing, and you should upgrade your whole 
+system, if possible.
+
+
+Accessories
+-----------
+Before beginning a video project, ensure there is enough storage space on your 
+computer to save all the necessary clips.  For example, one hour of 
+1080i video, such as from a mini-DV camcorder, takes up nearly 42 GB of 
+storage.  If your computer's internal storage device cannot store all the 
+clips, the solution is to buy an external drive.
+
+It would help if you had several cables, usually Firewire or USB, to connect 
```

Comment:
Again, this generalization is based on older-style cameras with physical mediums. Most video is now shot on SD cards, or downloaded from Phones or Cloud storage.